airflow_worker/Dockerfile

```
@@ -44,6 +44,6 @@ RUN cp ${AIRFLOW_HOME}/configuration/airflow.cfg* ${AIRFLOW_HOME}/.
 ENV AIRFLOW_HOME ${AIRFLOW_HOME}
 ENV TZ UTC
 ENV PYTHONPATH /opt/airflow
-ENV PYSTAC_STAC_VERSION_OVERRIDE=1.0.0
+ENV PYSTAC_STAC_VERSION_OVERRIDE=1.14.1
```

```suggestion
ENV PYSTAC_STAC_VERSION_OVERRIDE=1.0.0
```
this one is actually setting pystac to check that the stac metadata is stac version 1.0.0 so we want to keep it